Bottom: 90fa7f4e66d55b95b7a779e5026fe1a1d9df9b31
Top:    561ca6079291d397ced455b28cce0f5e6ef4a3a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-07 23:54:49 +0000

Refresh of hid-gtk-fix-prelight-state-for

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index a0f34d2..51ad6fd 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1612,7 +1612,7 @@ ghid_layer_buttons_color_update (void)
       lb = &layer_buttons[i];
 
       layer_process (&color_string, NULL, NULL, i);
-      active = gtk_toggle_button_get_active (lb);
+      active = gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (lb));
       layer_button_set_color (lb, color_string, active);
     }
 }
